### PR TITLE
Mahdiyeh/ fix: CFD related changes in DMT5 and Multiplier

### PR DESCRIPTION
**Pull request by mahdiyeh-fs**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Comment by sean-binary** [comment]
Hey @mahdiyeh-fs, this will need to be branched off Crowdin branch 🙂

**Comment by mahdiyeh-fs** [comment]
> Hey @mahdiyeh-fs, this will need to be branched off Crowdin branch

done

**Review by sean-binary — APPROVED** [review]

File: src/pages/trade-types/multiplier/_what-are-multipliers.js [hunk 1]
```diff
@@ -207,13 +231,14 @@ const WhatAreOptions = () => {
...
-                                    <QueryImage data={data['multiplier_no_multi_loss']} alt="tes" />
+                                    <QueryImage data={is_eu_country ? data['multiplier_no_multi_loss_eu'] : data['multiplier_no_multi_loss']}
+                                    alt="tes" />
                                 </ImgWrapper>
 
                                 <TextWrapper>
                                     <Text m="1.6rem 0 4rem 0">
                                         <Localize
-                                            translate_text="<0>With an equivalent $100 position on CFDs</0>, with 1:500 leverage, you risk 2% * $50,000 = <1>$1,000 loss</1>."
+                                            translate_text={is_eu_country ? "<0>With an equivalent $100 position on CFDs</0>, with 1:5 leverage, you risk 25% * $500 = <1>$125 loss</1>." : "<0>With an equivalent $100 position on CFDs</0>, with 1:500 leverage, you risk 2% * $50,000 = <1>$1,000 loss</1>."}
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_what-are-multipliers.js:241]: I don't think you're gonna be able to extract both the strings here for the translation. You can keep the `is_eu_country` check outside the Localize component. In this case, we're gonna need 2 components for either of the cases.

File: src/pages/trade-types/multiplier/_what-are-multipliers.js [hunk 2]
```diff
@@ -222,15 +247,15 @@ const WhatAreOptions = () => {
...
+                                        data={is_eu_country ? data['multiplier_with_multi_loss_eu'] : data['multiplier_with_multi_loss']}
                                         alt="tes"
                                     />
                                 </ImgWrapper>
 
                                 <TextWrapper>
                                     <Text mt="1.6rem">
                                         <Localize
-                                            translate_text="<0>With a x500 multiplier</0>, if the market goes down 2%, you'll <1>lose only $100</1>. An automatic stop out kicks in if your loss reaches your stake amount."
+                                            translate_text={is_eu_country ? "<0>With a x5 multiplier</0>, if the market goes down 25%, you'll <1>lose only $100</1>. An automatic stop out kicks in if your loss reaches your stake amount." : "<0>With a x500 multiplier</0>, if the market goes down 2%, you'll <1>lose only $100</1>. An automatic stop out kicks in if your loss reaches your stake amount."}
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_what-are-multipliers.js:258]: Same here.

File: src/pages/trade-types/multiplier/_what-are-multipliers.js (hunk 1 above)
Comment (mahdiyeh-fs) [reply src/pages/trade-types/multiplier/_what-are-multipliers.js]: fixed, thanks

**Review by sean-binary — APPROVED** [review]

**Pull request merged**